Project.toml

```
@@ -1,7 +1,7 @@
 name = "JuliaFormatter"
 uuid = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
 authors = ["Dominique Luna <dluna132@gmail.com>"]
-version = "1.0.39"
+version = "1.0.40"
```

This PR adds a new option `short_circuit_to_if`, so it would be better to bump the minor version.

```suggestion
version = "1.1.0"
```

---

by default it's off so from the user perspective there's no change implying there's not really a need to bump it. Also there's this compatibility thing with VSCode where bumping the minor version frequently leads to things being out of sync so it's easier to just bump the patch version.

---

>by default it's off so from the user perspective there's no change implying there's not really a need to bump it.

Hmm, Julia packages should respect semantic versioning, so I believe bumping the patch version should imply a bug fix. (cf. [Pkg.jl docs](https://pkgdocs.julialang.org/v1/compatibility/#Version-specifier-format), [semver docs](https://semver.org/#spec-item-6))

>Also there's this compatibility thing with VSCode where bumping the minor version frequently leads to things being out of sync so it's easier to just bump the patch version.

What is the reason behind this? I couldn't find the `[compat]` table in `Project.toml` files in the [julia-vscode](https://github.com/julia-vscode/julia-vscode) repository.

---

>Also there's this compatibility thing with VSCode where bumping the minor version frequently leads to things being out of sync so it's easier to just bump the patch version.

Is it because of this (https://github.com/julia-vscode/julia-vscode/issues/3226#issuecomment-1451803980)? If so, the problem about the JuliaFormatter's version in vscode is not related to bumping patch or minor version.